sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -253,6 +253,19 @@ abstract class JdbcDialect extends Serializable with Logging{
       }
     }
 
+    override def visitAggregateFunction(
+        funcName: String, isDistinct: Boolean, inputs: Array[String]): String = {
+      if (isSupportedFunction(funcName)) {
+        super.visitAggregateFunction(
+          dialectFunctionName(funcName).getOrElse(funcName), isDistinct, inputs)
+      } else {
+        throw new UnsupportedOperationException(
+          s"${this.getClass.getSimpleName} does not support aggregate function: $funcName");
+      }
+    }
+
+    protected def dialectFunctionName(funcName: String): Option[String] = None
```

Comment:
shall we apply this in `visitSQLFunction`?

Reply:
OK